code/items/IBuyableItem.cs

```diff
@@ -4,13 +4,18 @@
 
 namespace TTTReborn.Items
 {
-    public struct ShopItemData
+    public class ShopItemData
     {
-        public string Name;
-        public string Description;
-        public int Price;
-        public SlotType SlotType;
-        public Type Type;
+        public string Name { get; set; }
+        public string Description = "";
```

Comment:
Any reason why one is property and the other one isn't?

Reply:
Just Name and Price should be included into json. That's why. Otherwise, I had to do { internal get; internal set } which wouldn't make any difference